fl4health/losses/deep_mmd_loss.py

```diff
@@ -0,0 +1,224 @@
+from typing import Optional, Tuple
+
+import numpy as np
+import torch
+
+
+class ModelLatentF(torch.nn.Module):
+    """Latent space for both domains."""
+
+    def __init__(self, x_in: int, H: int, x_out: int):
+        """Init latent features."""
+        super(ModelLatentF, self).__init__()
```

Comment:
I know this isn't all your code, but this can be changed to `super().__init__()`